### DISCUSSION TITLE
something wrong with  Maxim Baranov's git configuration?

### DISCUSSION THREAD

**Discussion by laf0rge** [edgecomllc/eupf#505] [General]
Forgive me for this slightly strange question, but when reviewing the commit log, I see many commits with  the description of "f".  Is that some kind of misconfiguration of the git client that drops the commit message and always writes a bogus "f" character?

**Comment by pirog-spb** [comment]
Hi Harald @laf0rge 

You're right. We usually make PR's using squash&merge, but something definitely went wrong :)
Will pay closer attention to this. 
Thanks for your hint!

-- BR Alex
